packages/bootstrap/scss/_variables.scss

```diff
@@ -986,6 +986,33 @@ $scrollview-navigation-default-opacity: .7 !default;
 $scrollview-navigation-hover-opacity: 1 !default;
 $scrollview-navigation-hover-span-bg: null !default;
 
+// Switch
+
+$switch-size: 6em !default;
+$switch-border-radius: $switch-size !default;
+$switch-padding-x: null !default;
+$switch-padding-y: null !default;
+
+$swtich-track-size: null !default;
+$switch-container-border-width: 1px !default;
+$switch-container-padding-x: 2px !default;
+$switch-container-padding-y: $switch-container-padding-x !default;
+
+$switch-handle-size: (.4 * $switch-size) !default;
+$switch-handle-border-radius: $switch-size !default;
+$switch-handle-border-width: 1px !default;
+
+$swtich-on-handle-left: null !default;
+$swtich-off-handle-left: null !default;
+
+$switch-label-display: inline !default;
+$switch-label-spacing: 8px !default;
+
+$switch-occupied-space: add-three($switch-container-border-width, $switch-container-padding-x, $switch-handle-size);
```

Comment:
@elena-gancheva @joneff 
The layout is pretty much finalized. Could you check these three variables, and let me know if you see a better way to these calculations?